### PR TITLE
Add attributes panel

### PR DESCRIPTION
**Pull request by yuxitang-amzn**
*Issue #, if available:*

*Description of changes:*
Add attributes panel.

Screenshot:
Mockup:
image

Implementation:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/panels/internal-user-edit/attribute-panel.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+export function AttributePanel(props: {
+  state: UserAttributeStateClass[];
+  setState: Dispatch<SetStateAction<UserAttributeStateClass[]>>;
+}) {
+  const { state, setState } = props;
+  return (
+    <PanelWithHeader
+      headerText="Attributes - optional"
```
Comment (tianleh) [inline public/apps/configuration/panels/internal-user-edit/attribute-panel.tsx:105]: From the mockup, "optional" has a different font

File: public/apps/configuration/panels/internal-user-edit/attribute-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+              />
+            </FormRow>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <FormRow headerText={arrayIndex === 0 ? 'Value' : ''}>
+              <EuiFieldText onChange={onValueChangeHandler('value')} placeholder="Type in value" />
+            </FormRow>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFormRow hasEmptyLabelSpace={arrayIndex === 0 ? true : false}>
```
Comment (tianleh) [inline public/apps/configuration/panels/internal-user-edit/attribute-panel.tsx:82]: Just curious why hasEmptyLabelSpace is needed. From the mockup, looks like there is header text regardless of the array index.

Comment (yuxitang-amzn) [reply public/apps/configuration/panels/internal-user-edit/attribute-panel.tsx:82]: It adds a spacer here for vertical alignment:
image

File: public/apps/configuration/panels/internal-user-edit/attribute-panel.tsx (hunk 1 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/internal-user-edit/attribute-panel.tsx]: Updated, didn't cancel the bold since it is controlled by the EuiTitle.

**Pull request merged**